src/Avalonia.Controls/Window.cs

```diff
@@ -216,22 +217,25 @@ public void Close(object dialogResult)
         /// <summary>
         /// Hides the window but does not close it.
         /// </summary>
-        public void Hide()
+        public override void Hide()
         {
             using (BeginAutoSizing())
             {
                 PlatformImpl.Hide();
```

Comment:
Hmm, I've already done that - do you mean like this? https://github.com/AvaloniaUI/Avalonia/pull/935/files#diff-029f85158bbec909294403faa0014074R258. Or am I misunderstanding?